README.md

```diff
@@ -22,240 +22,286 @@ $ go get -u github.com/census-instrumentation/opencensus-go
+```
 
 ### To collect data for on-demand retrieveal
 Even if a view is registered, if it has no subscriber no data for it is collected. In order to retrieve data on-demand for view, either the view needs to have at least 1 subscriber or the libray needs to be instructed explicitly to collect collect data for the desired view.
 
-    // explicitly instruct the library to collect the view data for an on-demand retrieval.
-    if err := stats.ForceCollection(myView1); err != nil {
-        // handle error
-    }
-    ...
-
-    // Retrieve data. The returned rows are a []stats.Row. Each row is a struct:
-    //  {
-	//      Tags             []tags.Tag
-	//      AggregationValue AggregationValue
-    //  }
-    rows, err := stats.RetrieveData(v View)
-    if err != nil {
-        // handle error
-    }
-    ...
-
-    // explicitly instruct the library to stop collecting the view data for the
-    // on-demand retrieval. Has no impact on subscriptions, and if the view
-    // still has subscribers, the data for the view will still keep being
-    // collected.
-    if err := stats.StopForcedCollection(myView1); err != nil {
-        // handle error
-    }
-    ...
-
-## Recording usage/measurements
+```go
+// explicitly instruct the library to collect the view data for an on-demand retrieval.
+if err := stats.ForceCollection(myView1); err != nil {
+    // handle error
+}
+
+// Retrieve data. The returned rows are a []stats.Row. Each row is a struct:
+//  {
+//      Tags             []tags.Tag
+//      AggregationValue AggregationValue
+//  }
+rows, err := stats.RetrieveData(v View)
+if err != nil {
+    // handle error
+}
+
+// explicitly instruct the library to stop collecting the view data for the
+// on-demand retrieval. Has no impact on subscriptions, and if the view
+// still has subscribers, the data for the view will still keep being
+// collected.
+if err := stats.StopForcedCollection(myView1); err != nil {
+    // handle error
+}
+```
+
+### To record usage/measurements
 Recording usage can only be performed against already registered measure and and their registered views. Measurements are implicitly tagged with the tags in the context:
-    
-    // mi is a *MeasureInt64 and v is an int64 .
-    stats.RecordInt64(ctx, mi, v)
-
-    // mf is a *RecordFloat64 and v is an float64 .
-    stats.RecordFloat64(ctx, mf, v)
 
-    // multiple measurements can be performed at once.
-    stats.Record(ctx, mi.Is(4), mf.Is(10.5))
+```go
+// mi is a *MeasureInt64 and v is an int64 .
+stats.RecordInt64(ctx, mi, v)
+// mf is a *RecordFloat64 and v is an float64 .
+stats.RecordFloat64(ctx, mf, v)
+// multiple measurements can be performed at once.
+stats.Record(ctx, mi.Is(4), mf.Is(10.5))
+```
+
+### To retrieve collected data for a View
+
+```go
+// assuming c1 is the channel that was used to subscribe to myView1
+go func(c chan *stats.ViewData) {
+    for vd := range c {
+        // process collected stats received. The type of vd is
```

Comment:
You don't have to explain the type of vd. We are using a statically typed language, the type is clear to the user :)

Reply:
I admit this is redundant. Removed it.